src/v1/main.py

```diff
@@ -200,7 +200,7 @@ def format_time_label(hours: float) -> str:
         colorscale="Viridis",
         colorbar_title="Power [%]" if normalized else "Power [GW]",
         marker_opacity=0.5,
-        hovertemplate="<b>%{customdata}</b><br>Power: %{z:.2f} GW<extra></extra>",
+        hovertemplate="<b>%{customdata}</b><br>Power: %{z:.2f} %<extra></extra>",
```

Comment:
you need to condition this on `normalized`

so something like 
if normalized:
  hovertemplate="<b>%{customdata}</b><br>Power: %{z:.2f} %<extra></extra>"
else:
 hovertemplate="<b>%{customdata}</b><br>Power: %{z:.2f} GW<extra></extra>"
```
and then use `hovertemplate`